filament/include/filament/Engine.h

```diff
@@ -187,6 +187,59 @@ class UTILS_PUBLIC Engine {
     static Engine* create(Backend backend = Backend::DEFAULT,
             Platform* platform = nullptr, void* sharedGLContext = nullptr);
 
+#if UTILS_HAS_THREADING
+
+    /**
+     * A callback used with Engine::createAsync() called once the engine is initialized and it is
```

Comment:
Maybe mention that the callback will be invoked on an arbitrary worker thread and that the app should handle this with care.

Reply:
It's particularly important since they have to call getEngine() from the same thread as createAsync() so the app will need to do a bit of thread hopping here.